packages/codemods/src/codeMods/mods/PersonaToAvatar/PersonaToAvatar.mod.ts

```diff
@@ -0,0 +1,179 @@
+import {
```

Comment:
I don't think this kind of file should exist. If we wanted to rename 'Foo' to 'Bar' in the future, I would not want a dev copy pasting this file. 

I think we want to have something more like:

codemods.config.js:
```
export default const config = {
  actions: [
    {
       type: 'rename',
       from: {
         importPaths: [ 'office-ui-fabric-react', 'office-ui-fabric-react/lib/Persona', @fluentui/react', '@fluentui/react/lib' ],
         name: 'PersonaCoin',
       },
       to: {
         path: '@fluentui/react',
         name: 'Avatar'
       }
    }
  ]
};
```

Or if there is a way to make it even simpler, awesome. But we need to keep it the least error prone system for doing simple API surface updates. That way we can easily review codemods and have full confidence they will work without some simple mistake.

Reply:
@dzearing I agree, we won't have a file like this. This was a proof of concept mainly. I want to get the utility functions nailed down. I'll get this refactored for an example in the future.